controller.py

```diff
@@ -284,9 +284,6 @@ def find_and_get_url(links, rel, allowed_response_codes=None):
             create_method_kwargs=dict(stage=Library.REGISTERED)
         )
 
-        library.name = auth_document.title
-        library.description = auth_document.service_description
-
```

Comment:
I'd like to move a lot more code out of here and into `AuthenticationDocument.update_librar()`, but the base URL and the possibly mocked `do_get` make it a little tricky, so I didn't do much.